app.py

```diff
@@ -148,10 +149,13 @@ def get_data(ticker, range=0.05, maxSize=32, minVolumePerc=0.01):
     vol_grp_bid = vol_grp_bid[
         ((vol_grp_bid[TBL_VOLUME] >= minVolume) & (vol_grp_bid['count'] >= 2.0) & (vol_grp_bid['count'] < 70.0))]
     vol_grp_bid['unique'] = vol_grp_bid.index.get_level_values(TBL_VOLUME)
-    vol_grp_bid['unique'] = vol_grp_bid['unique'].apply(round_sig, args=(3,))
+    vol_grp_bid['unique'] = vol_grp_bid['unique'].apply(round_sig, args=(1,2))
```

Comment:
'Unique' contains the exact order size, wich needs 3 significant because of distuingtion beetween 1.5 wich is human behaviour and 2.536 wich typically is a bot

So I disagree with this line change